src/preview.ts

```diff
@@ -43,6 +40,16 @@ export class Preview extends TypedEventEmitter<Preview.Events> {
     await win.load(location)
     this.emit('open', win, location)
 
+    // Override close function to ignore raising error if a target page has already close
+    const { close } = win
+    win.close = async () => {
+      try {
+        return await close.call(win)
+      } catch (e) {
+        if (!e.message.includes('Target closed.')) throw e
```

Comment:
Even if raising error `Target closed.`, the `close` event has called correctly. We decided to ignore this error.